src/nrncvode/spt2queue.cpp

```diff
@@ -255,7 +255,7 @@ double TQueue::q2least_t() {
 	return 1e50; // must be larger than any possible t
 }
 
-#include <spaux.c>
-#include <sptree.c>
-#include <spdaveb.c>
+#include <spaux.cpp>
```

Comment:
why are those headers included with angle brackets? why are we compiling them with `-iSystem` ?

Reply:
Not sure. To be tackled with refactoring.